docs/workflows/api-token.md

```diff
@@ -68,14 +68,7 @@ Use the following curl examples below to obtain an authorization code and then c
 1. Authenticate to obtain an authorization code:
 
 <MultiLanguageCodeBlock
-curl='curl -X GET "https://auth-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/applications/$(APPLICATION_ID)/authorize" \
--d "response_type=code" \
--d "client_id=$(APPLICATION_CLIENT_ID)" \
--d "redirect_uri=$(REDIRECT_URI)" \
--d "scope=openid" \
--d "state=$(STATE)" \
--d "code_challenge=$(OPTIONAL_CODE_CHALLENGE)" \
--d "code_challenge_method=S256"'
+curl='curl -X GET "https://auth-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/applications/$(APPLICATION_ID)/authorize?response_type=code&client_id=$(APPLICATION_CLIENT_ID)&redirect_uri=$(REDIRECT_URI)&scope=openid&state=$(STATE)&code_challenge=$(OPTIONAL_CODE_CHALLENGE)&code_challenge_method=S256"'
```

Comment:
Is possible to break it up with `\`  just so that the curl command is more readable? It looks like there's another curl command in this file that could use that too if you don't mind doing that as well?

Reply:
> Is possible to break it up with \ just so that the curl command is more readable?

Great idea! I googled a little bit and found this syntax:

```bash
curl -v -G "https://auth-us.beyondidentity.com/v1/tenants/a/realms/b/applications/c/authorize" \
--data-urlencode "response_type=code" \
--data-urlencode "client_id=d" \
--data-urlencode "redirect_uri=e" \
--data-urlencode "scope=openid" \
--data-urlencode "state=f" \
--data-urlencode "code_challenge=g" \
--data-urlencode "code_challenge_method=S256"
```

I'll fix it on monday.

> It looks like there's another curl command in this file that could use that too

Sure, I'll update the token listing examples to use this syntax too.

Reply:
thank you!